app/src/main/res/values/strings.xml

```diff
@@ -202,17 +201,17 @@
         &lt;br/&gt;
         &lt;h5&gt;
 
-        Open Source License
+        Copylefted Libre License
```

Comment:
```suggestion
        Copyleft Libre License
```

Reply:
_Copylefted_ qualifies the "libre". Otherwise it has to be "Copyleft-libre license" I think.